apps/project/filter_set.py

```diff
@@ -260,22 +257,20 @@ def filter_title(self, qs, _, value):
             return qs
         return qs.filter(title__icontains=value)
 
-    def filter_is_entry_less_than(self, qs, _, value):
-        if value is None:
-            return qs
-        qs = qs.annotate(
-            entry_count=models.functions.Coalesce(models.Subquery(
-                Entry.objects.filter(
-                    project=models.OuterRef('id')
-                ).order_by().values('project').annotate(
-                    count=models.Count('id', distinct=True)
-                ).values('count')[:1],
-                output_field=models.IntegerField()
-            ), 0)
-        )
+    def filter_exclude_entry_less_than(self, qs, _, value):
```

Comment:
`def filter_is_test(self, qs, _, value):`
Use `qs` instead of `Project.objects`